### ISSUE TITLE
Specific email not receiving emails Investigation of <robert_turtoiu@yahoo.com> [bz1417862]

### ISSUE THREAD

**Issue by bug-mirror** [mozilla/fxa-bugzilla-mirror#413]
From https://bugzilla.mozilla.org/show_bug.cgi?id=1417862

Created attachment 8928950
Untitled.png

User: Robert
robert_turtoiu@yahoo.com
I think that this is what it says to type here(Not to good at english)

The user has tried the following:
Nothing actually, because i don't know what to try. Its just blocked. 

The email is working, for aprox 5 or 6 years actually.

I noticed that is not an abnormal problem among the firefox accounts, i really need the account to work again.
Simply put i am too lazy to make a new one and login to different sites and remember the password and yada yada.

If you need something else from my behalf well you know where to find me.
